pkg/project/util.go

```diff
@@ -175,3 +181,27 @@ var intersection = apply(func(bs []Equatable, a Equatable) bool {
 var difference = apply(func(bs []Equatable, a Equatable) bool {
 	return !contains(bs, a)
 })
+
+var getBoolEnvVar = func(key string, fallback bool) (bool, error) {
+	value, exists := os.LookupEnv(key)
+	if exists {
+		boolValue, err := strconv.ParseBool(value)
+		if err == nil {
+			return boolValue, nil
+		} else {
+			return fallback, fmt.Errorf("WARN: environment value of %s should be of type Boolean. Setting to a fallback value %t", key, fallback)
```

Comment:
Putting the word `WARN` doesn't make it any less of an error to go.

Reply:
Why even bother with returning `error` when https://github.com/jfrog/terraform-provider-project/pull/32/files#diff-78bdb51875e934e7c55ab887da3d8d2afcd35bc21d4684815ead7cf26392c844R228 ignores the error.

Reply:
removed the error return as we are not going to throw error to CLI